kornia/color/hls.py

```diff
@@ -105,7 +105,7 @@ def forward(self, image: torch.Tensor) -> torch.Tensor:  # type: ignore
         return rgb_to_hls(image)
 
 
-def rgb_to_hls(image):
+def rgb_to_hls(image, eps=1.19209e-07):
```

Comment:
Typing

Reply:
Is this the only exact value that matches opencv ?